autogpt_platform/backend/backend/blocks/example/example.py

```diff
@@ -0,0 +1,143 @@
+import logging
+from typing import Any
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema, BlockType
+from backend.data.model import (
+    APIKeyCredentials,
+    ContributorDetails,
+    CredentialsField,
+    SchemaField,
+)
+
+from ._api import ExampleClient
+from ._auth import TEST_CREDENTIALS, TEST_CREDENTIALS_INPUT, ExampleCredentialsInput
+
+logger = logging.getLogger(__name__)
+
+
+class ExampleBlock(Block):
+
+    class Input(BlockSchema):
+        name: str = SchemaField(
+            description="The name of the example block", placeholder="Enter a name"
+        )
+        greetings: list[str] = SchemaField(
+            description="The greetings to display", default=["Hello", "Hi", "Hey"]
+        )
+        is_funny: bool = SchemaField(
+            description="Whether the block is funny", placeholder="True", default=True
+        )
+        # Only if the block needs credentials
+        credentials: ExampleCredentialsInput = CredentialsField(
+            description="The credentials for the example block"
+        )
+
+    class Output(BlockSchema):
+        response: dict[str, Any] = SchemaField(
+            description="The response object generated by the example block."
+        )
+        all_responses: list[dict[str, Any]] = SchemaField(
+            description="All the responses from the example block."
+        )
+        greeting_count: int = SchemaField(
+            description="The number of greetings in the input."
+        )
+        error: str = SchemaField(description="The error from the example block")
+
+    def __init__(self):
+        super().__init__(
+            # The unique identifier for the block, this value will be persisted in the DB.
+            # It should be unique and constant across the application run.
+            # Use the UUID format for the ID.
+            id="380694d5-3b2e-4130-bced-b43752b70de9",
+            # The description of the block, explaining what the block does.
+            description="The example block",
+            # The list of contributors who contributed to the block.
+            # Each contributor is represented by a ContributorDetails object.
+            contributors=[ContributorDetails(name="Craig Swift")],
```

Comment:
Seems like a contributor needs to be in a centralized place.

Reply:
I don't really see the value of the contributors field. 
I feel it maybe better managed based on the git history. 

I'm happy to remove it if its not best practice.

What do you think?